index.html

```diff
@@ -5,10 +5,95 @@
+            </div>
+        </div>
+
+        <div class="right">
+            <div class="dummy_text">
+                <p>This is not a real online service! Don't input your actual information or password. If you do,
+                    nothing will actually happen. It doesn't get submitted anywhere.
+                </p>
+                <p>I don't know. I'm not the boss of you. Do whatever.</p>
+            </div>
+
+            <form action="#" method="post">
+                <div class="form_content">
+                    <div class="sign-up_txt"><p>Sign up</p></div>
+                    <div class="top_row">  
+                        <div>
+                            <label for="first_name">First Name</label>
+                            <input type="text" id="first_name" name="first_name" placeholder="First Name" required>
+                        </div>
+
+                        <div>
+                            <label for="last_name">Last Name</label>
+                            <input type="text" id="last_name" name="last_name" placeholder="Last Name" required>
+                        </div>
+
+                        <div>
+                            <label for="user_email">Email*</label>
+                            <input type="email" id="user_email" name="user_email" placeholder="email@example.com" required>
+                        </div>
+                    </div>
+                    <div class="bottom_row">
+                        <div>
+                            <label for="phone_number">Phone Number</label>
+                            <input type="tel" id="phone_number" name="phone" placeholder="(###)-###-####">
+                        </div>  
+
+                        <div>
+                            <label for="password">Password*</label>
+                            <input type="password" id="password" name="password" 
+                                pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}" 
+                                title="Must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters"
+                                placeholder="********"
+                                required>
+                        </div>
+                        
+                        <div>
+                            <label for="password">Confirm Password*</label>
+                            <input type="password" id="password_confirm" name="confirm_password" oninput="check(this)"
+                                pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}" 
+                                placeholder="********"
+                                required>
+                        </div>
+                    </div>
+
+                    <p class="password_msg_req">*Password must contain at least one number and one uppercase and lowercase letter, and at least 8 or more characters)</p>
+                    
+                    <div class="btn">
```

Comment:
Hmm - I don't like this. 

This isn't a button but it has a class called "btn"